src/nrc/api/tasks.py

```diff
@@ -35,16 +49,23 @@ def deliver_message(sub_id: int, msg: dict, **kwargs) -> None:
             headers={"Content-Type": "application/json", "Authorization": sub.auth},
         )
         response_init_kwargs = {"response_status": response.status_code}
+        if response.status_code != 204:
```

Comment:
Not sure if this is too strict, but when registering a webhook, there is validation to explicitly check for 204's (https://github.com/open-zaak/open-notificaties/blob/main/src/nrc/api/validators.py#L40)

Reply:
I'd relax this to any `2xx` status code, 204 is correct from the standard, but I'd like to get the standard changed so that 200, 201, 202, 204 are all valid. Some may be relevant for asynchronous processing when they store the notification locally or put it on their on message bus for example.

Reply:
Should I also relax this constraint in the `CallbackURLValidator`?

Reply:
yeah, seems appropriate. Couldn't find anything in the textual standard that describes this, other than the API spec of the consumer documenting a HTTP 204